sentry-config.cmake.in

```diff
@@ -2,19 +2,34 @@
 
 set(SENTRY_BACKEND @SENTRY_BACKEND@)
 set(SENTRY_TRANSPORT @SENTRY_TRANSPORT@)
+set(SENTRY_BUILD_SHARED_LIBS @SENTRY_BUILD_SHARED_LIBS@)
+set(SENTRY_LINK_PTHREAD @SENTRY_LINK_PTHREAD@)
 
 if(SENTRY_BACKEND STREQUAL "crashpad")
-	if(@SENTRY_CRASHPAD_SYSTEM@)
+    set(SENTRY_CRASHPAD_SYSTEM @SENTRY_CRASHPAD_SYSTEM@)
+	if(SENTRY_CRASHPAD_SYSTEM)
 		find_package(crashpad REQUIRED)
 	else()
 		include("${CMAKE_CURRENT_LIST_DIR}/sentry_crashpad-targets.cmake")
 	endif()
 endif()
 
+if(SENTRY_BACKEND STREQUAL "breakpad" AND NOT SENTRY_BUILD_SHARED_LIBS)
+    set(SENTRY_BREAKPAD_SYSTEM @SENTRY_BREAKPAD_SYSTEM@)
+	if(SENTRY_BREAKPAD_SYSTEM)
+		find_package(PkgConfig REQUIRED)
+		pkg_check_modules(BREAKPAD REQUIRED IMPORTED_TARGET breakpad-client)
+	endif()
+endif()
+
 include("${CMAKE_CURRENT_LIST_DIR}/sentry-targets.cmake")
 
-if(SENTRY_TRANSPORT STREQUAL "curl" AND NOT @BUILD_SHARED_LIBS@)
+if(SENTRY_TRANSPORT STREQUAL "curl" AND (NOT @BUILD_SHARED_LIBS@ OR NOT SENTRY_BUILD_SHARED_LIBS))
 	find_package(CURL REQUIRED)
 	set_property(TARGET sentry::sentry APPEND
 		PROPERTY INTERFACE_LINK_LIBRARIES ${CURL_LIBRARIES})
 endif()
+
+if(SENTRY_LINK_PTHREAD AND NOT SENTRY_BUILD_SHARED_LIBS)
+    find_package(Threads REQUIRED)
```

Comment:
Why is threads required?

Reply:
Because we use a background thread for the transport.